### ISSUE TITLE
black 'include_trailing_comma' not supported

### ISSUE THREAD

**Issue by CagtayFabry** [ryantam626/jupyterlab_code_formatter#206]
**Checklist prior to opening an issue**
- [x] I have followed fully the installation steps laid out in the documentation site.
- [x] I have restarted jupyterlab.
- [x] I have read the FAQ section in the documentation site.

**Describe the bug**
The default black option `"include_trailing_comma": true,` raises the following error on a new install when running the code formatter (clicking the icon in juypterlab):

```
Jupyterlab Code Formatter Error

__init__() got an unexpected keyword argument 'include_trailing_comma'
```

I tested on a clean conda install:
```
conda create -n cformat python=3.8 jupyterlab=3.0 black=20.8b1 isort=5.8.0 jupyterlab_code_formatter=1.4.7
```

Removing the setting in the code formater config solves this.
Maybe remove or comment out this option in the default config?

**Comment by ryantam626** [comment]
My bad, it's not a valid option to begin with, have fixed it now with 1.4.8

**Comment by CagtayFabry** [comment]
thanks !
awesome extension btw 🚀
